### DISCUSSION TITLE
Could not find com.trustwallet:wallet-core:2.9.5

### DISCUSSION THREAD

**Discussion by FatherOfLove** [trustwallet/wallet-core#2335] [Q&A]
In the android studio, I want to add the trustwallet dependency to build.gradle which I follow exactly the same as https://developer.trustwallet.com/wallet-core/integration-guide/android-guide#adding-library-dependency, 
below is my build gradle files
![image](https://user-images.githubusercontent.com/22650614/174948778-1878db9c-83e6-4094-9ce5-63619f0d1a1c.png)
![image](https://user-images.githubusercontent.com/22650614/174949047-72d7449a-3d53-45f1-b582-4ae17561ac3c.png)

And I get below error when I build the project: Could not find com.trustwallet:wallet-core:2.9.5
![image](https://user-images.githubusercontent.com/22650614/174949136-5b53a57a-314e-45ef-8702-7c762064843e.png)

**Comment by hewigovens** [comment]
Did you add `gpr.user` and `gpr.key` to local.properties or setup `GITHUB_USER` and `GITHUB_TOKEN` env variables ?

**Reply by FatherOfLove** [reply]
Yes, I added gpr.user and gpr.key to local.properties file, and it still showing the same error
![image](https://user-images.githubusercontent.com/22650614/175196276-34569376-545c-4dc2-9f7a-b298506fb7b9.png)

**Reply by hewigovens** [reply]
I tried the sample in wallet core and it works with 2.9.5, try set the username and password directly here https://github.com/trustwallet/wallet-core/blob/master/samples/android/build.gradle#L22 to see if that works for you

**Reply by FatherOfLove** [reply]
Should I get some permission from you guys to run this script?

**Reply by hewigovens** [reply]
no, also make sure your github token has the permission to read packages

**Reply by FatherOfLove** [reply]
I was able to run the sample trustwallet. But when I integrated into my project, it was failing.

**Reply by FatherOfLove** [reply]
I solved this problem by adding the maven to settings.gradle file
![image](https://user-images.githubusercontent.com/22650614/175793266-99166814-142e-42bf-b77c-4a05e6d2eacd.png)
